### ISSUE TITLE
Request for Guidance on Labeling Data for PlanetScope Imagery

### ISSUE THREAD

**Issue by zhangalex** [DPIRD-DMA/OmniCloudMask#12]
Hi Nick and thanks for your work on OmniCloudMask! It's an amazing project, I did some testing on my local, it's great.

We're planning to contribute to this project using PlanetScope imagery instead of Sentinel-2. Since the model was trained on Sentinel-2, we understand the data characteristics may differ.

To help us get started with training or fine-tuning on PlanetScope, could you please provide some guidance or a brief outline on the labeling process you used for generating cloud masks? Specifically:

- What format are the labels in, polygon or points?
- How were they generated (manual annotation, automated tools, etc.)? do we need a vast number of labelled features?
- Any tools or workflows you'd recommend for labeling PlanetScope data? currently, we are using QGIS.

Thanks

Best regards,

Alex

**Comment by wrignj08** [comment]
Hi Alex, I'm glad you are enjoying using OCM. I'm not sure if you are aware but OCM does actually work pretty well on PlanetScope data already, you just need to re-sample the 3 m data to 10 m and pass the RG+NIR bands to OCM. Similar to the [Maxar notebook example](https://github.com/DPIRD-DMA/OmniCloudMask/blob/main/Maxar%20example.ipynb) in the project repo. If you would like I can add a PlanetScope example, let me know.

If you do still want to fine-tune the models for PlanetScope then I suggest you take a look at the [training](https://github.com/DPIRD-DMA/OmniCloudMask/tree/main/training) folder in the repo. It shows you how to download the training dataset and walks through training the models. You could follow that and include some additional PlanetScope data if you want. 

If you do end up making a training dataset it would be great if you could share it to help improve OCM for everyone else!

We did not collect the training data for this project, instead we used the amazing [CloudSEN12](https://cloudsen12.github.io) dataset. You can check out the [paper](https://www.nature.com/articles/s41597-022-01878-2) for details on data collection.

Thanks,

Nick

**Comment by wrignj08** [comment]
Hi Alex, 

No problems, I have added a [PlanetScope example](https://github.com/DPIRD-DMA/OmniCloudMask/blob/main/examples/PlanetScope.ipynb) in the new examples [folder](https://github.com/DPIRD-DMA/OmniCloudMask/tree/main/examples).

Please let me know how this works for you, take note that depending on the PlanetScope product type you may need to change the PS_band_order, it is currently setup for the 4 band PlanetScope product.

Have fun,

Nick
